### ISSUE TITLE
Room options "Require moderator approval before joining" (waiting room) and "All users join as moderators" can be chosen at the same time

### ISSUE THREAD

**Issue by bwildenhain** [bigbluebutton/greenlight#2951]
Room creators might expect that participants will have to be approved before they get into the meeting with moderator rights. In reality, they'll first get moderator rights and will enter directly into the meeting room. To avoid confusion both options should be mutually exclusive.

**Comment by farhatahmad** [comment]
Please note: Greenlight v3 has been released. With this new version, many of the issues and bugs that were present in v2 have been resolved.

As a result, we will no longer be providing updates or support for v2 (except for major security issues), and we will be closing any outstanding bug reports / feature requests related to v2. While we understand that some of you may still be using v2, we highly encourage you to upgrade to v3 to take advantage of the improved features and stability. If your request/bug still applies to v3, please open a new issue for it
